### DISCUSSION TITLE
General cFS Papers for Citation?

### DISCUSSION THREAD

**Discussion by mdicicco-hbr** [nasa/cFS#285] [Q&A]
Does anyone in the community know of a good general paper covering cFS that would be a good citation if we want to reference cFS in a publication?  I looked on ntrs.nasa.gov and there are a lot of presentations but no overview papers.

**Comment by skliper** [comment]
Somewhat dated but provides an overview: https://cfs.gsfc.nasa.gov/cFS-OviewBGSlideDeck-ExportControl-Final.pdf
More up-to-date and contains much of the same overview information but training focused: https://ntrs.nasa.gov/citations/20205000691

**Reply by mdicicco-hbr** [reply]
Thanks for the suggestion.  I was hoping for a small summary paper along the lines of the famous Quigley ICRA paper on ROS.  This will probably work for us in terms of citing something related to cFS, but it would have been good to have quotable text and a nice succinct description to use for trying to explain what cFS is all about.  I'll keep an eye out.

**Comment by dmccomas** [comment]
If you're specifically looking for papers here are some options. The first option is free and the next 2 are professional societies that will charge if you're not a member. 

1.  The Small Satellite Conference maintains an archive of papers/presentations at https://digitalcommons.usu.edu/smallsat/ It has a search engine and tells you how to cite papers from the archive.  

2.  IEEE, Institute of Electrical and Electronics Engineers,  has a search engine at https://ieeexplore.ieee.org/Xplore/home.jsp. They have a convenient "cite this" button that generates a popup with the info.

3. AIAA, American Institute of Aeronautics and Astronautics, has a search engine at https://arc.aiaa.org/search/advanced

**Reply by dmccomas** [reply]
My initial wording may be a little confusing. IEEE and AIAA charge if you want a complete copy of a paper. Accessing their sites  and searching for publications/abstracts is publically available.

**Reply by astrogeco** [reply]
Thanks for the answer Dave! Also, feel free to email authors directly for a copy of the paper. Most of them will provide it free of charge.

**Comment by szemerick** [comment]
We describe cFS in a section of our NOS3 paper: https://arxiv.org/abs/1901.07583
It's not an overall summary, but it may serve as a citation reference for a cFS introduction.

**Comment by astroesteban** [comment]
GitHub actually just [added support for citations](https://github.blog/2021-08-19-enhanced-support-citations-github/). This is done by adding a _CITATION.cff_ file in the project's root directory. It lets a user quickly get the APA or BibTeX formatted citation directly from this repo.

Here is an example:
```yaml
cff-version: 1.2.0
 message: "If you use this software, please cite it as below."
 authors:
 - family-names: "LastName"
   given-names: "FirstName"
 title: "NASA Core Flight Software"
 url: "https://github.com/nasa/cfs" 
```

This is something that could be added to the project to make this easier in the future.
